actgen/change_q.py

```diff
@@ -127,6 +138,7 @@ def run(self):
                              n_outputs=n_outputs,
                              n_hidden_layers=self.params['n_hidden_layers'],
                              n_units_per_layer=self.params['n_units_per_layer'],
+                             device=self.params['device'],
```

Comment:
See comment about `MLP([...]).to(device)`

Reply:
I think this one still might need to know the device because some tensors in `directed_qnet.py` still needs to know which device to be put on.